### ISSUE TITLE
OpenFAST can't find tail fin aerodynamics file (when absolute)

### ISSUE THREAD

**Issue by mayankchetan** [OpenFAST/openfast#2359]
**Bug description**
<A clear and concise description of the bug.>
OpenFAST gives a `segfault` or `Allocatable argument 'fileinfo' is not allocated` when `TFinFile` is set as an absolute path.

**To Reproduce**
<Update the following list with your specific information.>
1. Compile with `any compiler`
2. Run `Tailfin_FreeYaw1DOF_PolarBased` case with `TFinFile` set as an absolute path
3. See the error

**Expected behavior**
<A clear and concise description of what you expected to happen.>
Be able to handle absolute path.

**Screenshots, if applicable**
<Add screenshots to help explain your problem.>
![image](https://github.com/user-attachments/assets/b579bd90-f902-451f-9155-c32de088f4ac)

**OpenFAST Version**
<Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:>

```
 **************************************************************************************************
 OpenFAST

 Copyright (C) 2024 National Renewable Energy Laboratory
 Copyright (C) 2024 Envision Energy USA LTD

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

 OpenFAST-3.5.TEST-1457-g93eeaa47-dirty
 Compile Info:
  - Compiler: GCC version 11.3.0
  - Architecture: 64 bit
  - Precision: double
  - OpenMP: No
  - Date: Aug  7 2024
  - Time: 03:42:12
 Execution Info:
  - Date: 08/07/2024
  - Time: 04:57:34+0000
```

**System Information (please complete the following information):**
 - OS: `Ubuntu 22.04.2 LT` Docker on Mac OS `13.6.8 (22G820)` 
 - Compiler: GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0
 - Compiler settings: `-DCMAKE_BUILD_TYPE=Debug`

**Additional context**
<Add any other context about the problem here.>

**Comment by deslaughter** [comment]
@mayankchetan Can this issue be closed since #2360 was merged?

**Comment by mayankchetan** [comment]
Yes!
